src/client/Microsoft.Identity.Client/Platforms/Features/WamBroker/WamBroker.cs

```diff
@@ -711,6 +711,13 @@ private async Task<MsalTokenResponse> AcquireTokenSilentDefaultUserPassthroughAs
         {
             var transferToken = await _msaPassthroughHandler.TryFetchTransferTokenSilentDefaultAccountAsync(authenticationRequestParameters, defaultAccountProvider).ConfigureAwait(false);
 
+            if (string.IsNullOrEmpty(transferToken))
+            {
+                throw new MsalUiRequiredException(
+                    MsalError.InteractionRequired,
+                    "Found an MSA account, but could not retrieve a transfer token for it.");
```

Comment:
Maybe you want to mention WAM in the string?

```suggestion
                    "Found an MSA account, but could not retrieve a transfer token for it when calling WAM.");
```

Would it be useful to have an aka.ms link to describe what the developer should do?

Reply:
There is nothing the developer can do, except to pop-up UI via AcquireTokenInteractive. I'll update the error message.